dev/update_changelog.py

```diff
@@ -143,7 +143,7 @@ def main(prev_version, release_version, remote):
     author_to_prs = defaultdict(list)
     unlabelled_prs = []
     for pr in prs:
-        if pr.author == "mlflow-automation":
+        if pr.author in ["mlflow-automation", "mlflow-app[bot]"]:
```

Comment:
Do we still keep mlflow-automation here?

Reply:
I don't think we need it. Let me update.